compiler/rustc_query_impl/Cargo.toml

```diff
@@ -7,6 +7,8 @@ edition = "2021"
 
 
 [dependencies]
+memoffset = { version = "0.6.0", features = ["unstable_const"] }
+field-offset = "0.3.5"
```

Comment:
This creates 3 versions of `memoffset` (0.6, 0.7, 0.8). Is there particular reason to use 0.6 version instead of 0.7 or 0.8?